packages/flutter_tools/test/general.shard/android_plugin_test.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Renamed from other file since it only contained Android. Added test cases to prove that embedding version identification works